### DISCUSSION TITLE
WebServer API: How to retrieve the result of "... then[X]else[Y]]" filter using "WebServer API" calls?

### DISCUSSION THREAD

**Discussion by msameen99** [TiddlyWiki/TiddlyWiki5#6875] [Q&A] [answered]
Hello,

I’m using the [WebServer API: Get All Tiddlers](https://tiddlywiki.com/#WebServer%20API%3A%20Get%20All%20Tiddlers) successfully to retrieve my Tiddlers using customized filters.

My Question, Is it possible to use "WebServer API" to execute/return a filter ended with `[then[X]else[Y]]` where "X" and "Y" are just text (not tiddler's titles)?

Note: 
currently I'm using a working workaround to accomplish this task as the following:
For each filter results (X,Y, Yes, No, .....), I'm creating "dummy" tiddlers with titles are exactly the same as the expected results , but this mean I need to create as many tiddlers as my possible results.

Regards

**Comment by Jermolene** [comment, marked as answer]
Hi @msameen99 I don't think it is currently possible to retrieve filter results that are not actual tiddlers. But it is a useful capability, so I'd be in favour of introducing a mechanism that makes it possible. Perhaps another optional parameter.

**Reply by msameen99** [reply]
Thanks a lot Jermolene for your answer,
